lib/games/games_module.rb

```diff
@@ -17,19 +16,44 @@ def add_game
     print 'Please state how many years ago it was published: '
     published_date = gets.chomp
     new_game = Game.new(title, multiplayer, last_played_at, published_date)
-    @game_list.push(new_game)
+    @game_list.push(title: new_game.title, multiplayer: new_game.multiplayer, last_played_at: new_game.last_played_at, published_date: new_game.published_date)
```

Comment:
I know that working with JSON for recreating instances of classes is tricky and (maybe?) a bad way to do it. But think in this situation: 

If needed how would you call a method that exists in that Game? Or How would you retrieve a relation of that Game Item with an Author class? 

That's why I would suggest you change that part so you'd create the instance of the class again when you are reading that file. 

Check out this [article](https://www.sitepoint.com/choosing-right-serialization-format/) that explains how you can re-instantiate classes that were saved in JSON format.

Reply:
Great advice, now I did it working with the class and creating it again when reading the file 👍